hoverkite-protocol/src/lib.rs

```diff
@@ -0,0 +1,131 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use core::ops::RangeInclusive;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum Side {
+    Left,
+    Right,
+}
+
+impl Side {
+    pub fn opposite(self) -> Self {
+        match self {
+            Self::Left => Self::Right,
+            Self::Right => Self::Left,
+        }
+    }
+}
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum Command {
+    // FIXME: stop using RangeInclusive, so we can derive Copy
+    SetMaxSpeed(RangeInclusive<i16>),
+    SetSpringConstant(u16),
+    SetTarget(i64),
+    Recenter,
+    BatteryReport,
```

Comment:
`ReportBattery` would fit better with the other commands, which all start with an imperative.